### PR TITLE
[DUOS-668][risk=no] Remove the Manage Ontologies and Add Ontologies p…

### PR DESCRIPTION
**Pull request by Marianie-Alphonse**
-Remove ManageOntologies.js and AddOntologiesModal.js
-Remove lines containing OntologiesModal from AdminConsoleBox.css, AdminConsole.js, Routes.js

### CODE REVIEW

**Review by Marianie-Alphonse — COMMENTED** [review]
Should I edit the Ajax file?

**Comment by rushtong** [comment]
@Marianie-Simeon 

> Should I edit the Ajax file?

Yes, please do. I think it's just:
* Files.getOntologyFile
* Ontology.postOntologyFile
* Ontology.retrieveIndexedFiles
* Ontology.deleteOntologyFile
* Ontology.getOntologyTypes
* Ontology.guid

But please do find usages to ensure that.

**Comment by Marianie-Alphonse** [comment]
I just updated the ajax.js

File: src/libs/ajax.js [hunk 1]
```diff
@@ -296,12 +296,6 @@ export const DAR = {
     return darInfo;
   },
 
-  translateDataUse: async dataUse => {
```
Comment (rushtong) [inline src/libs/ajax.js]: This method in use elsewhere, please revert this deletion.

File: src/libs/ajax.js [hunk 2]
```diff
@@ -371,12 +365,6 @@ export const DAR = {
     return await res.json();
   },
 
-  getAutoCompleteOT: async partial => {
```
Comment (rushtong) [inline src/libs/ajax.js]: This method in use elsewhere, please revert this deletion.

**Review by rushtong — CHANGES_REQUESTED** [review]
Need a couple of reverts (see comments inline) but otherwise looks good.

**Comment by Marianie-Alphonse** [comment]
It should be updated now

**Review by rushtong — APPROVED** [review]
👍  Thank you!

**Pull request merged**